components/esp_https_ota/src/esp_https_ota.c

```diff
@@ -411,7 +438,8 @@ esp_err_t esp_https_ota(const esp_http_client_config_t *config)
     esp_https_ota_handle_t https_ota_handle = NULL;
     esp_err_t err = esp_https_ota_begin(&ota_config, &https_ota_handle);
     if (https_ota_handle == NULL) {
-        strncpy(ota_perform_err_str, "INVALID OTA HANDLE", MAX_OTA_PERFORM_ERROR_MSG_LENGTH);
```

Comment:
maybe we can check that `ota_perform_err_str` actually has a value? if it does, then we don't set it (as you have here) but if it has no value (ie maybe somehow those other strings didn't get set) then we use this generic error string

Reply:
Every place where a failure can occur in esp_https_ota_begin(), the error string is updated and the esp_https_ota_begin() sets the handle parameter to NULL and returns an ESP error code.  The ota_perform_err_str will always have a string in it if a failure occurs. I verified this before I removed the loading of the "INVALID_OTA_HANDLE".

BUT

To protect against a future change that may cause a new failure and not set the string, it would be better to see the "INVALID OTA HANDLE" rather than an empty string in the job notifications.  I will put the the line I removed back in.

Reply:
yeah, having a base case error message i think is critical in case anything else happens to be added / some other change occurs without knowledge of this.

Reply:
The change has been made and PR submitted.